Cartfile.resolved

```diff
@@ -1,12 +1,12 @@
-github "thoughtbot/Curry" "v3.0.0"
+github "Carthage/Commandant" "0.11.3"
+github "Carthage/ReactiveTask" "96e1013e0a6782bf0faba27485fe2d0a1153b531"
 github "Quick/Nimble" "v6.0.1"
-github "jdhealy/PrettyColors" "6f54504f8715c5d51114bcff821c48fd87d8a8b7"
 github "Quick/Quick" "v1.0.0"
+github "ReactiveCocoa/ReactiveSwift" "1.0.0"
 github "antitypical/Result" "3.1.0"
-github "thoughtbot/Runes" "v4.0.1"
+github "jdhealy/PrettyColors" "6f54504f8715c5d51114bcff821c48fd87d8a8b7"
 github "jspahrsummers/xcconfigs" "0.9"
-github "thoughtbot/Argo" "v4.1.2"
-github "Carthage/Commandant" "0.11.3"
-github "ReactiveCocoa/ReactiveSwift" "1.0.0"
-github "Carthage/ReactiveTask" "96e1013e0a6782bf0faba27485fe2d0a1153b531"
 github "mdiep/Tentacle" "0.5.0"
+github "thoughtbot/Argo" "v4.1.2"
+github "thoughtbot/Curry" "v3.0.0"
+github "thoughtbot/Runes" "v4.0.1"
```

Comment:
Sure, this was just the output from running a carthage update to double check the versions were good. Since the versions are the same i'll revert.